packages-proprietary/tokens/src/components/ams/switch.tokens.json

```diff
@@ -7,10 +7,12 @@
       "inline-size": { "value": "3.5rem" },
       "thumb": {
         "background-color": { "value": "{ams.color.background}" },
-        "inline-size": { "value": "1.75rem" },
         "block-size": { "value": "1.75rem" },
+        "border": { "value": "{ams.border.width.s} solid transparent" },
```

Comment:
Maybe we shouldn't change this component. In both the new and old design, a `box-shadow` is more appropriate imo. And we already fixed this component in forced colors mode.

Reply:
Alright. Also saves us from a glitchy hover effect that it introduced. Reverted, but maintained using a border width token and sorting ‘inline-size’ correctly.